webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/Table/Table.js

```diff
@@ -113,12 +113,20 @@ export const Table = ({
         url={url}
         refreshData={refreshData}
       />
-      <PFTable variant="compact" ouiaId="table" isStriped>
+      <PFTable
+        variant="compact"
+        ouiaId="table"
+        isStriped
+        isExpandable={childrenOutsideTbody}
+      >
         <Thead>
           <Tr ouiaId="table-header">
             {showCheckboxes && (
               <Th aria-label="checkbox-header" key="checkbox-th" />
             )}
+            {childrenOutsideTbody && (
+              <Th aria-label="expansion-carat" key="expansion-carat" />
+            )}
```

Comment:
If we use this, instead of the column adjustment that I suggested in Katello, please open a PR in REX to remove remove the extra Th from there

Reply:
Will do :+1:

Reply:
ping

Reply:
Sorry for the delay :) https://github.com/theforeman/foreman_remote_execution/pull/972